src/main/java/it/gov/pagopa/pu/organization/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package it.gov.pagopa.pu.organization.exception;
+
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springdoc.api.ErrorMessage;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+public class ControllerExceptionHandler {
+
+  @ExceptionHandler(ResourceNotFoundException.class)
+  @ResponseStatus(value = HttpStatus.NOT_FOUND)
+  public ErrorMessage resourceNotFoundException(ResourceNotFoundException ex, HttpServletRequest request) {
```

Comment:
logging the HttpStatus we are duplicating its value, let's set it once
```suggestion
  public ResponseEntity<ErrorMessage> resourceNotFoundException(ResourceNotFoundException ex, HttpServletRequest request) {
       HttpStatus httpStatus = HttpStatus.NOT_FOUND;
       ...
       return ResponseEntity
                      .status(httpStatus)
                      .body(new ErrorMessage("resource not found: %s".formatted(ex.getMessage())));
```

Reply:
done